Source/Core/DolphinQt/Config/ToolTipControls/BalloonTip.cpp

```diff
@@ -315,7 +315,8 @@ void BalloonTip::UpdateBoundsAndRedraw(const QPoint& target_arrow_tip_position,
   // Place the arrow tip at the target position whether the arrow tip is drawn or not
   const int target_balloontip_global_x =
       target_arrow_tip_position.x() - static_cast<int>(arrow_tip_x);
-  const int rightmost_valid_balloontip_global_x = screen_rect.width() - size_hint.width();
+  const int rightmost_valid_balloontip_global_x =
+      screen_rect.left() + screen_rect.width() - size_hint.width();
   // If the balloon would extend off the screen, push it left or right until it's not
   const int actual_balloontip_global_x =
       std::max(0, std::min(rightmost_valid_balloontip_global_x, target_balloontip_global_x));
```

Comment:
```suggestion
  const int actual_balloontip_global_x =
      std::max(screen_rect.left(), std::min(rightmost_valid_balloontip_global_x, target_balloontip_global_x));
```

I think screens to the left of the primary screen need this change to show up properly.

Reply:
Yes, your suggestion helps keep things on the correct screen for the most part.